File: old-0.0/fizzbuzz/src/scala/fizzbuzz.scala
object fizzbuzz {
  def main(args:Array[String]):Unit = {
    for (n <- 1 to 50) {
      if (n % 3 == 0 && n % 5 == 0) {
        println("fizz buzz")
      } else if (n % 3 == 0) {
        println("fizz")
      } else if (n % 5 == 0) {
        println("buzz")
      } else {
        println(n)
      }
    }
  }
}

File: old-0.0/fizzbuzz/src/ocaml/fizzbuzz.ml
for n = 1 to 50 do
  match n mod 3, n mod 5 with
  | 0, 0 -> Printf.printf "fizz buzz\n"
  | 0, _ -> Printf.printf "fizz\n"
  | _, 0 -> Printf.printf "buzz\n"
  | _, _ -> Printf.printf "%d\n" n
done
